.github/workflows/build-windows.yaml

```diff
@@ -18,18 +18,20 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Enable Corepack
```

Comment:
These `Enable Corepack` should be after `Use Node.js`.

As it is currently done, it is doing the `corepack enable` using the corepack from the default nodejs (which looks to be 18.19.1 currently), then it will be replacing corepack and nodejs.